consensus/polybft/state_store_exit_event.go

```diff
@@ -54,11 +62,17 @@ type ExitEventStore struct {
 }
 
 // initialize creates necessary buckets in DB if they don't already exist
-func (s *ExitEventStore) initialize(tx *bolt.Tx) error {
+func (s *ExitEventStore) initialize(tx *bolt.Tx, blockNumber uint64) error {
 	if _, err := tx.CreateBucketIfNotExists(exitEventsBucket); err != nil {
 		return fmt.Errorf("failed to create bucket=%s: %w", string(exitEventsBucket), err)
 	}
 
+	if forkmanager.GetInstance().IsForkEnabled(chain.DoubleSignSlashing, blockNumber) {
```

Comment:
What happens if the fork is not yet activated on node startup, but it exists in code? Bucket for `slashingExitEventsBucket` will not be created. Do we need to worry about this?

Reply:
Per @Nemanja0x comment, we have decided to remove checking of the hard fork and create a slashing bucket unconditionally.
https://github.com/0xPolygon/polygon-edge/pull/1823/commits/429357f227843802a249e8fdb494b950bdfb638c